com/views.py

```diff
@@ -268,6 +270,34 @@ def get_context_data(self, **kwargs):
         return super().get_context_data(**kwargs) | {"date": self.object.dates.first()}
 
 
+class NewsFeed(Feed):
+    title = _("News")
+    link = reverse_lazy("com:news_list")
+    description = _("All incoming events")
+
+    def items(self):
+        return (
+            NewsDate.objects.filter(
+                news__is_moderated=True,
+                end_date__gte=timezone.now() - (relativedelta(months=6)),
+            )
+            .prefetch_related("news")
```

Comment:
```suggestion
            .select_related("news")
```

Il y a une news par date, donc il faut select_related, ici